tests/lib/bin/invariant-tool

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+if [ $# -eq 0 ]; then
+	echo "usage: invariant-tool check [INVARIANT...]"
```

Comment:
Nitpick, I think this should also show available invariants, it's not clear what the allowed arguments are.

Reply:
Done.